.github/workflows/release.yaml

```diff
@@ -6,9 +6,9 @@ on:
             release-version:
               type: string
               required: true
-              description: 'Version number (for example: 0.1.0)'
-            is-latest:
-              description: 'Select if the built image should be tagged as latest'
+              description: 'Version number (for example: v0.1.0)'
+            is-stable:
+              description: 'Select if the built image should be tagged as stable'
```

Comment:
where do you use is-stable ?

Reply:
Good question. I just updated the existing workflow without giving it a ton of thought. your following suggestion is a good one.